docs/OntologiesReviewWorkflow

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc
+title: Ontology Review Process
+---
+
+This page deals with the process, policies, and guidelines for manually reviewing newly-submitted ontologies, with particular focus on operational aspects such as:
+
+- which ontologies are reviewed, and in what order?
+- how are reviewers chosen?
+- what is the workflow for conducting a review?
+- what are the criteria used?
+
+Ontologies are reviewed manually after completing the automated evaluation. Though the automated evaluation checks for adherence to [OBO Foundry principles](http://www.obofoundry.org/principles/fp-000-summary.html), it cannot capture certain nuances outlined in the principles nor evaluate ontology quality. The purpose of the manual review is to check the ontology for issues that the NOR Dashboard does not cover.
+
+# Steps prior to manual review
+
+Once an ontology is submitted to the OBO Foundry, it is added to the New Ontology Dashboard for automated evaluation. The steps involved in ontology submission and subsequent dashboard evaluation are described in detail in this [FAQ answer](http://obofoundry.org/faq/how-do-i-register-my-ontology.html) and links therein.
+
+# Review priority
+
+All submitted ontologies will be manually reviewed based on the order in which the ontology becomes compliant with the automated evaluation (that is, when the [New Ontology Request (NOR) Dashboard](http://obofoundry.org/obo-nor.github.io/dashboard/index.html) status is set to 'pass').
+
+# Choosing reviewers
+
+Reviewers are chosen on a rotating basis from the members of the [OBO Foundry Operations Committee](http://obofoundry.org/docs/Membership.html).
+
+# Ontology review workflow
+
+The manual process begins once a reviewer is assigned. The reviewer will assess key aspects of the ontology (see below) and report the findings both to the submitter (via the issue tracker ticket used for submission) and to the OBO Operations Committee. The latter will discuss the findings and make a recommendation for acceptance, revision, or rejection.
+
+# Manual review criteria
+
+Criteria for review include (but are not limited to):
+
+1. Ontology scope - Do the terms fall within the ontology's stated target domain of knowledge? Was the ontology developed for a very specific purpose or community?
```

Comment:
> Was the ontology developed for a very specific purpose or community?

But that's ok, right? That just makes it a domain ontology or whatever we ended up calling it?

Reply:
Correct, it is okay. The question is evaluated because the answer impacts certain other aspects of the review.